README.md

```diff
@@ -224,6 +229,10 @@ List of \<services\>
 - tadp  
 - tp  
 - rule-901  
```

Comment:
This is also why I'd prefer the rule-executer branch to reflect the rule processor id and not the rule-executer; it's a bit confusing having the .env branch variable say one thing, and then the docker-compose.yaml saying another...